### PR TITLE
GetListRaw - added list separator arg

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39652

## Description
Add a separator argument (the default is `,` )

## Screenshots
with default sep:
image
on the list:
image

with custom sep:
image
on the list:
image

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by abaumgarten** [comment]
@nicolkaz-panw FYI

File: Packs/CommonScripts/ReleaseNotes/1_4_11.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### GetListRow
+- Added **list_separator** argument to enable custom list delimiter.
+- Updated the Docker image to: *demisto/python3:3.8.3.9324*.
```
Comment (bakatzir) [inline Packs/CommonScripts/ReleaseNotes/1_4_11.md:5]:
```suggestion
- Added the **list_separator** argument to enable custom list delimiter.
- Updated the Docker image to: *demisto/python3:3.8.3.9324*.
```

File: Packs/CommonScripts/Scripts/GetlistRow/GetListRow.py [hunk 2]
```diff
@@ -79,8 +79,9 @@ def main():
     parse_all = args['parse_all']
     header = args.get('header', '')
     value = args.get('value', '')
+    list_separator = args.get('list_separator', ',')
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/GetlistRow/GetListRow.py:82]:
```suggestion
    list_separator = args.get('list_separator', ',') or ','
```

File: Packs/CommonScripts/Scripts/GetlistRow/GetListRow.py [hunk 3]
```diff
@@ -17,8 +17,8 @@ def validate_header_exists(headers, header):
         return_error("Error: The supplied header name was not found.")
 
 
-def list_to_headers_and_lines(list_data):
-    lines_and_headers = [line.split(',') for line in list_data.split('\n')]
+def list_to_headers_and_lines(list_data, list_separator):
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/GetlistRow/GetListRow.py:20]:
```suggestion
def list_to_headers_and_lines(list_data, list_separator: str):
```

File: Packs/CommonScripts/Scripts/GetlistRow/GetListRow.py [hunk 4]
```diff
@@ -54,7 +54,7 @@ def parse_relevant_rows(headers, lines, header, value, context, parse_all=False)
     )
 
 
-def parse_list(parse_all, header, value, list_name):
+def parse_list(parse_all, header, value, list_name, list_separator):
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/GetlistRow/GetListRow.py:57]:
```suggestion
def parse_list(parse_all, header, value, list_name, list_separator: str):
```

File: Packs/CommonScripts/Scripts/GetlistRow/GetListRow.yml [hunk 5]
```diff
@@ -20,14 +22,22 @@ args:
...
   isArray: false
   name: parse_all
   predefined:
   - 'True'
   - 'False'
   required: false
   secret: false
+- default: false
+  defaultValue: ','
+  description: Seperator for splitting the list by
```
Comment (bakatzir) [inline Packs/CommonScripts/Scripts/GetlistRow/GetListRow.yml:36]:
```suggestion
  description: Separator to split the list by.
```

**Review by bakatzir — COMMENTED** [review]
please add given/when/then to the UT.
also - where you able to produce some errors in the split upon certain delimiters?
I want to make sure that customers will get a meaningful error in case it happens.

**Review by bakatzir — APPROVED** [review]

**Pull request merged**